discovery-provider/src/tasks/index_solana_plays.py

```diff
@@ -434,14 +434,14 @@ def process_solana_plays(solana_client_manager: SolanaClientManager, redis):
                     ): tx_sig
                     for tx_sig in tx_sig_batch
                 }
-                for future in concurrent.futures.as_completed(parse_sol_tx_futures):
-                    try:
+                try:
```

Comment:
need to move this try outside the futures function because timeout error is thrown at that level, not inside the .result()